src/Faithlife.Utility/StringUtility.cs

```diff
@@ -78,6 +78,38 @@ public static string ReplaceOrdinal(this string source, string oldValue, string
 #endif
 		}
 
+#if NETSTANDARD2_0
```

Comment:
When the comment correctly describes the _meaning_ of the method, then we no longer need this `#if` block: the description of the method shouldn't change between target frameworks.